spec/chewy/type/adapter/active_record_spec.rb

```diff
@@ -63,7 +65,10 @@
   describe '#import' do
     def import(*args)
       result = []
-      subject.import(*args) { |data| result.push data }
+      subject.import(*args) do |data|
+        result.push data
+        yield if block_given?
+      end
       result
     end
```

Comment:
Have you meant `subject.to_enum(:import, *args).each(&block).to_a`.

It is super-fancy, like lot of code here :wink:.